src/layouts/Layout.astro

```diff
@@ -22,27 +22,34 @@ const { title, headerProps } = Astro.props;
     <meta name="viewport" content="width=device-width" />
     <link
       rel="icon"
-      href="/website-v2/favicon.svg"
+      href="/website-v2/footer/logo.png"
       sizes="any"
       type="image/svg+xml"
     />
     <meta name="generator" content={Astro.generator} />
     <title>{title}</title>
   </head>
   <body class="bg-white">
-    <nav
-      class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.95] p-4"
-    >
-      <div class="m-auto flex w-full justify-between">
-        <a href="/website-v2">
-          <img
-            src="/website-v2/footer/logo.png"
-            class="h-16"
-          />
-        </a>
-        <Hamburger client:only />
-      </div>
-    </nav>
+    {
+      headerProps.isContain ? (
+        <div />
+      ) : (
+        <div>
+          <nav class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.95] p-4">
+            <div class="m-auto flex w-full justify-between">
+              <a href="/website-v2">
+                <img
+                  src="/website-v2/footer/logo.png"
+                  class="h-16"
+                />
+              </a>
+              <Hamburger client:only />
+            </div>
+          </nav>
+        </div>
```

Comment:
nACK on removing nav bar. Lets remove the header and put some margin top so that teh image is below nav bar